### ISSUE TITLE
Add Support for Custom Ore Generation: First Revival

### ISSUE THREAD

**Issue by TomeWyrm** [hilburn/NotEnoughResources#40]
Pretty much exactly what is says in the title. I figured it should be on the issue tracker as an enhancement so it'll be a bit more obvious if/when it's fixed :-)

**Comment by warezit** [comment]
I also agree this needs to be a thing... I checked around the interwebs, because I thought I read somewhere that this was implemented already, but I don't believe it is yet. It sounds like hilburn is aware of it, and working towards it... Some links (albeit, every comment I can find about this is from 4+ months ago):

http://www.reddit.com/r/feedthebeast/comments/2mkybg/preview_not_enough_resources/
> Currently we are working on compatibility with CoFHCore to automatically register any ore generation defined using their system, and an API to allow other modders to register their own ores, mobs and dungeon loot with our system easily.

http://www.reddit.com/r/feedthebeast/comments/2lmjmk/mod_idea_mob_drops_index/clyypb5
> At the moment I'm working on hacking CoFH Core to pull any custom oregen that's been set there - once we have that and vanilla working we're gonna try to get some sort of reasonable API up and running so people can add things

http://www.reddit.com/r/feedthebeast/comments/2mkybg/preview_not_enough_resources/cm604cb
> It will take data out of cofh core which most people use to tweak ore generation automatically. Everything else will be reliant on modders to utilise our API

http://www.reddit.com/r/feedthebeast/comments/2mkybg/preview_not_enough_resources/cm5ehse
> Thanks guys for letting me see snippets of your CoFH-Core hacking in-progress..

Thanks for your work on this hilburn! We'll be patient!

**Comment by warezit** [comment]
Edit: CoFH Core Custom Ore Gen is already supported:

https://github.com/hilburn/NotEnoughResources/issues/20#issuecomment-65895591
> NER is almost entirely compatible with COFH oregen - it doesn't yet handle biome an dimension restriction but that is coming soon. The others have been added to the list
